weaviate-services/agents/transformation-agent-get-started.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<blockquote>we have to decide which&nbsp;<code>collection</code>&nbsp;it may have accesss to.</blockquote><p>Wdyt about saying "we have to let the agent know which collection it is to modify".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/weaviate/recipes/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>